### DISCUSSION TITLE
Sharing formatting configuration across team

### DISCUSSION THREAD

**Discussion by alex529** [redhat-developer/vscode-yaml#877] [Q&A] [answered]
What is the recommended way of sharing formatting rules across the entire team?
From what i see all of the settings here are done thru the user/workspace settings, which aren't ideal

**Comment by gorkem** [comment, marked as answer]
The formatter for the `vscode-yaml` is a delegate to the prettier. Unfortunately, `vscode-yaml` does not read the prettier configuration files (`.prettierrc` and friends). So my recommendation is to disable the formatter from the extension and use prettier for formatting directly.
